### PR TITLE
Added automatic loading of symbols from local file

### LINKED ISSUE #4668: Loading symbols/debugging information via gdb://protocol
Need feature: When I debug windows binary with winedbg/gdb I don't see list of functions and strings after `aaa` command:

```
[0x7b45f70c] iz

[0x7b45f70c] afl
0x7b45f700 14 1 fcn.7b45f700
0x7b460730 24 1 eip
```

### PR DESCRIPTION
**Pull request by SrimantaBarua**
Partially fixes #4668 
Fails in case file is not locally present at same abspath

### CODE REVIEW

File: binr/radare2/radare2.c [hunk 1]
```diff
@@ -805,6 +805,11 @@ int main(int argc, char **argv, char **envp) {
 						optind--; // take filename
 					}
 					fh = r_core_file_open (&r, pfile, perms, mapaddr);
+					if (!strcmp (debugbackend, "gdb") && fh && fh->desc
```
Comment (radare) [inline binr/radare2/radare2.c]: Move to a separate function if (isGdbUri() or so

File: libr/io/p/io_gdb.c [hunk 2]
```diff
@@ -143,7 +143,13 @@ static RIODesc *__open(RIO *io, const char *file, int rw, int mode) {
 		} else if ((i_pid = desc->pid) < 0) {
 			i_pid = -1;
 		}
+		// Get name
+		char *name = gdbr_exec_file_read (desc, i_pid);
+		if (name) {
+			file = name;
+		}
 		riogdb = r_io_desc_new (&r_io_plugin_gdb, i_pid, file, rw, mode, riog);
+		free(name);
```
Comment (radare) [inline libr/io/p/io_gdb.c]: Missing space

**Comment by XVilka** [comment]
May be search also nearby? In the current directory e.g.?

**Comment by radare** [comment]
Maybe honor an eval var like the one to specify where the src is

**Comment by SrimantaBarua** [comment]
Current directory is done.
I see a `e file.location`, and grep suggests it's not used anywhere. Should I reuse that? It's probably better to specify file name than directory since local file name can be different.
Or something like `dbg.exe.path`?

**Pull request merged**